### PR TITLE
TEDEFO-2124 add OPT-060-Lot after OPT-070-Lot in NTD 7 to 22,38,39,40,CEI

### PR DESCRIPTION
**Pull request by rouschr**
It was branched from origin/update_sdk_1.7.0
And the destination is origin/update_sdk_1.7.0

### CODE REVIEW

**Comment by rouschr** [comment]
For PR see:
https://github.com/OP-TED/eForms-SDK/pull/425

File: .ted/tedweb/search-metadata.json
Comment (rousso) [inline .ted/tedweb/search-metadata.json:1]: This commit is not relevant to the ticket, so we need to remove it from this PR

File: notice-types/15.json [hunk 1]
```diff
@@ -1195,6 +1195,20 @@
...
           "description" : "Electronic Invoicing",
           "_label" : "field|name|BT-743-Lot"
+        }, {
+          "id" : "OPT-060-Lot",
+          "contentType" : "field",
+          "displayType" : "COMBOBOX",
+          "description" : "Execution Requirement Factor",
+          "_label" : "field|name|OPT-060-Lot",
+          "_presetValue" : "performance"
+        }, {
```
Comment (rousso) [inline notice-types/15.json:1205]: This is duplicate. Needs to be fixxed.

**Review by rousso — CHANGES_REQUESTED** [review]
Please fix the two comments first.

**Comment by rouschr** [comment]
I will close this PR as there was a commit that comes from another branch, it will be cleaner to redo this PR.

**Pull request closed without merging**